pom.xml

```diff
@@ -397,6 +404,11 @@
       <artifactId>jdbi3-guava</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.jdbi</groupId>
+      <artifactId>jdbi3-testing</artifactId>
```

Comment:
Can we add the test scope here or is it necessary for full builds?

Reply:
This was from an earlier strategy.  I was able to remove this dependency.